tests/examples/awful/wallpaper/corner1.lua

```diff
@@ -0,0 +1,30 @@
+--DOC_NO_USAGE --DOC_GEN_IMAGE
+local awful =  { --DOC_HIDE
+    wallpaper = require("awful.wallpaper"), --DOC_HIDE
+    placement = require("awful.placement"), --DOC_HIDE
+} --DOC_HIDE
+local wibox = require("wibox") --DOC_HIDE
+local beautiful = require("beautiful") --DOC_HIDE
+
+screen[1]._resize {x = 0, y = 0, width = 320, height = 196} --DOC_HIDE
+
+
+   awful.wallpaper {
+       screen = screen[1],
```

Comment:
Maybe it would be a good practice to first define the screen with something like `local s = screen[1] --DOC_HIDE` for example with only 1 screen. It would make the code more relatable to the user rc.lua and allow the code to work out of the box on multi monitor setups.

Reply:
done